react-components/src/hooks/useSkyboxFromScene.tsx

```diff
@@ -51,7 +51,8 @@ function initializeSkybox(
   texture: THREE.Texture,
   viewer: Cognite3DViewer
 ): [THREE.Object3D, () => void] {
-  const skyboxGeometry = new THREE.SphereGeometry(1000, 20, 20);
+  const skyboxRadius = 10;
```

Comment:
Are we sure radius of 10 units(meter) is sufficient to cover vary large models?

Reply:
It does not need to cover the model, it must just surround the camera. It has a render-order of -1 and disabled depth-write, so it is essentially just pasted to the screen before anything else